docs/topics/native/native-cocoapods-libraries.md

```diff
@@ -285,4 +285,25 @@ kotlin {
         }
     }
 }
+```
+
+### Share Kotlin cinterop between dependent Pods
+
+If you're adding multiple dependencies on Pods using the `pod()` function, you might encounter issues in Xcode
```

Comment:
It doesn't matter whether a build is from Xcode or not. Most probably, users will encounter it inside our IDE (AS/Intellij) first.

I would add some specifics here, like
```
// pod("WebImage") cinterop
fun image(): WebImage

// pod("Info") cinterop
fun printImageInfo(image: WebImage)

// user code
printImageInfo(image())
```

If a user doesn't set-up the correct dependencies between interops, then the code would be red because `WebImage` type would come from the different interops (and thus packages). If interop-dependencies are set up correctly, then the code would be green.

Reply:
Thank you! Removed mentions of Xcode and replaced the code sample